### PR TITLE
Update Hizo + plenty of desc additions

### PR DESCRIPTION
**Pull request by WarriorGallade**

### CODE REVIEW

File: data/mods/gen9ssb/moves.ts [hunk 1]
```diff
@@ -1984,46 +1991,6 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
-		// fake switch a la revival blessing
-		selfSwitch: true,
-		condition: {
-			duration: 1,
-			// reviving implemented in side.ts, kind of
-		},
-		secondary: null,
-		target: "self",
-		type: "Dark",
-	},
```
Comment (KrisXV) [inline data/mods/gen9ssb/moves.ts:2025]: ??

Comment (WarriorGallade) [reply data/mods/gen9ssb/moves.ts:2025]: it felt fine to remove the old ability since it's effectively unused code after the abiliy change

Comment (KrisXV) [reply data/mods/gen9ssb/moves.ts:2025]: you removed the move

Comment (WarriorGallade) [reply data/mods/gen9ssb/moves.ts:2025]: oh. am bad

**Pull request merged**